<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QMainWindow" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>문체부 제목 바탕체</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>문체부 제목 바탕체</family>
    </font>
   </property>
   <widget class="QTableWidget" name="stock_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NAME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRICE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PROFIT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>STOCK</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SALES</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="admin_back">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 232, 254);
font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <property name="text">
     <string>뒤로가기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stock_change">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 197, 197);
font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <property name="text">
     <string>수정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stock_delete">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 197, 197);
font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="money_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <row>
     <property name="text">
      <string>money</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>50000</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10000</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5000</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>1000</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>500</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>100</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="money_push">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 197, 197);
font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <property name="text">
     <string>입금</string>
    </property>
   </widget>
   <widget class="QPushButton" name="money_pop">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 197, 197);
font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <property name="text">
     <string>출금</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_ID">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>NAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_NAME"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>PRICE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_PRICE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>PROFIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_PROFIT"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>STOCK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_STOCK"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>SALES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_SALES"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>문체부 제목 바탕체</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;문체부 제목 바탕체&quot;;</string>
    </property>
    <property name="text">
     <string>재고관리</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="chef_burger_sales">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>sales</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>cheese burger</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>chicken burger</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>bulgogi burger</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>shrimp burger</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="profit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>221</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'문체부 제목 바탕체'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'문체부 제목 바탕체';&quot;&gt;순익&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
